src/components/ListPage/ListPage.ts

```diff
@@ -1,158 +1,139 @@
-import * as React from 'react';
-import { RouteComponentProps } from 'react-router';
-import * as MessageCenter from '../../utils/MessageCenter';
-import { Pagination } from '../../types/Pagination';
-import { ActiveFilter, FilterType } from '../../types/Filters';
+import history from '../../app/History';
 import { config } from '../../config';
+import { ActiveFilter, FilterType } from '../../types/Filters';
+import { Pagination } from '../../types/Pagination';
 import { SortField } from '../../types/SortFilters';
+import * as MessageCenter from '../../utils/MessageCenter';
 
 export namespace ListPage {
```

Comment:
Note that ListPage has changed several times over time, and maybe the name in inaccurate now.
Before, it was an abstract React Component.
Now it's turning into a Helper/Utility namespace. We could rename to reflect that change.